src/ripple/beast/core/SystemStats.cpp

```diff
@@ -61,24 +60,29 @@ getStackBacktrace()
         std::distance(std::begin(stack), std::end(stack)),
         stack, nullptr);
 
-    HeapBlock<SYMBOL_INFO> symbol;
-    symbol.calloc (sizeof (SYMBOL_INFO) + 256, 1);
-    symbol->MaxNameLen = 255;
-    symbol->SizeOfStruct = sizeof (SYMBOL_INFO);
+    // Allow symbols that are up to 1024 bytes long
+    std::size_t const nameLength = 1024;
+    std::size_t const nameSize = nameLength * sizeof(((SYMBOL_INFO *)0)->Name);
```

Comment:
I realize the c-cast of 0 is the idiomatic way of doing this, but I also tried these:

`
sizeof((static_cast<SYMBOL_INFO*>(0))->Name)
sizeof(SYMBOL_INFO{0}.Name)
`

and both worked on GCC...the MS compiler might disagree. I don't know if either one is better than the current expression. Perhaps a sizeof_member macro would be also be useful?